lua/orgmode/objects/edit_special/init.lua

```diff
@@ -90,16 +90,18 @@ function EditSpecial:init()
 
   self:_set_context(bufnr, ctx)
 
+  --config:setup_mappings('edit_src')
+
   utils.buf_keymap(
     bufnr,
     'n',
-    config.mappings.edit_src.org_edit_src_abort,
+    config.mappings.edit_src.org_edit_src_abort:gsub('<prefix>', config.mappings.prefix),
```

Comment:
Create a helper method in Config class for these `gsub('prefix')` changesand just call it here and in the setup_mappings.  Better to have it on single place

Reply:
done